README.md

```diff
@@ -41,49 +41,54 @@ The best part about [stamps](https://en.wikipedia.org/wiki/Stamp_%28object-orien
 
 ```js
 const mixin1 = {
-  componentDidMount() {
+  componentWillMount() {
     this.state.mixin1 = true;
   },
 };
 
 const mixin2 = {
-  componentDidMount() {
+  componentWillMount() {
     this.state.mixin2 = true;
   },
 };
 
-const component = stampit(React, {
+const Component = stampit(React, {
   state: {
     comp: false,
     mixin1: false,
     mixin2: false,
   },
 
-  someMethod() {
+  _onClick() {
+    return this.state;
+  },
+
+  componentWillMount() {
     this.state.comp = true;
   },
 
-  componentDidMount() {
-    this.someMethod();
+  render() {
+    return <input type='button' onClick={this._onClick.bind(this)} />;
   },
 }).compose(mixin1, mixin2);
+```
 
-const instance = component();
-instance.componentDidMount();
+```js
+shallowRenderer.render(<Component />);
+const button = shallowRenderer.getRenderOutput();
 
 assert.deepEqual(
```

Comment:
You should supply an english-language description of the feature you're testing for all assertions, especially in documentation. ;)